pandas/_libs/lib.pyx

```diff
@@ -107,18 +109,36 @@ def memory_usage_of_objects(arr: object[:]) -> int64_t:
 def is_scalar(val: object) -> bint:
     """
     Return True if given value is scalar.
+    
+    Parameters
+    ----------
+    val : numpy array scalar (e.g. np.int64), Python builtin numerics,
+          Python builtin byte arrays and strings, None,
+          instances of datetime.datetime, instances of datetime.timedelta,
+          Period, instances of decimal.Decimal, Interval, DateOffset,
+          Fraction, Number
+          The value to be checked.
+    
+    Returns
+    -------
+    True if the given value is scalar, False otherwise.
+    
+    Examples
+    --------
+    >>> dt = pd.datetime.datetime(2018,10,3)
```

Comment:
can you show an example with a list as well (to return False)

Reply:
sure! will do !